docs/networks/software-versions.mdx

```diff
@@ -20,15 +20,15 @@ Release candidates are software releases that are also released to the [Testnet]
 | --- | --- |
 | XDR | `v22.0` |
 | Rust XDR | `v22.0.0` |
-| Soroban Environment | `v22.0.0` |
+| Soroban Environment | `v22.1.0` |
 | Stellar Core | `v22.0.0` |
-| Soroban Rust SDK | `v22.0.0-rc.2.1` |
-| Stellar CLI | `TBD` |
-| Soroban RPC | `TBD` |
-| Stellar Horizon | `TBD` |
-| Stellar Quickstart | `TBD` |
-| Stellar JS Stellar Base | `TBD` |
-| Stellar JS Stellar SDK | `TBD` |
+| Soroban Rust SDK | `v22.0.0-rc.3.2` |
```

Comment:
This can be updated to [v22.0.0](https://github.com/stellar/rs-soroban-sdk/releases/tag/v22.0.0).